azkaban-common/src/main/java/azkaban/executor/ExecutorApiClient.java

```diff
@@ -31,6 +51,63 @@
 @Singleton
 public class ExecutorApiClient extends RestfulApiClient<String> {
 
+  final private static Logger logger = Logger.getLogger(ExecutorApiClient.class);
+
+  final private String DEFAULT_TRUSTSTORE_PATH = "keystore";
```

Comment:
Can you please make it static? Same for DEFAULT_TRUSTSTORE_PASSWORD.

Reply:
Good catch. Will fix.